packages/desktop-client/src/components/common/Toggle.css

```diff
@@ -0,0 +1,40 @@
+.react-switch-checkbox {
```

Comment:
Can this be declared on `Toggle.tsx` itself for consistency? I don't see any .css files anywhere else in the project.

Reply:
Good call. Thanks for catching it. I've updated it.